3_css/readme.md

```diff
@@ -86,13 +86,49 @@ Comments are intended for better code comprehension, this are pieces of the code
 }
 ```
 ## CSS basic selectors 
-    CSS selectors are used to select the HTML element that we want to style.
-    Basic selectors in CSS are the following:
+Selectors lets you modify individual elements in our HTML file. Here are some useful selector you should know:
     
-    1) Element selector: Selects HTML element using their name.
-    2) ID selector: Selects HTML element using their id attribute. Usage: Hash(#) followed by id.
-    3) Class selector: Selects HTML element using their class name. Usage: Period(.) followed by class name.
-    4) Universal selector: Selects all HTML elements in the page. Usage: Asterisk(*) is used as universal selector
```

Comment:
Please don't remove this content, merge your changes instead so that your contributions does not remove these changes.

Reply:
Hello Daniel! I followed your instructions. It's okay now? Let me know! Thank you.

Reply:
Thank you!